21.md

```diff
@@ -89,10 +92,12 @@ In a more particular case, when user is denominating the amount in `CURRENCY`, b
 
 In another particular case, when the user is denominating the amount in millisatoshi, but wants it to be converted to `CURRENCY` on `SERVICE`.
 ```diff
-- <callback><?|&>amount=<milliSatoshi>
-+ <callback><?|&>amount=<milliSatoshi>&convert=<CURRENCY>
+- <callback><?|&>amount=<millisatoshi>
++ <callback><?|&>amount=<millisatoshi>&convert=<CURRENCY>
 ```
 
+Note that the amount provided in all requests is always in integer number denominated in the smallets possible unit of the selected currency. The smallest unit need to be um sync with the `displayDecimals` parameter, so the `WALLET` has all the needed information to receive input and show output properly.
```

Comment:
Note that the amount provided in all requests is always *an* integer number denominated

Reply:
ab98b7d6ce1350dd4e2426eb110e7c6cf216c862